components/automate-ui/src/app/modules/policy/list/policy-list.component.html

```diff
@@ -29,7 +29,7 @@
                       <chef-table-cell class="type-column">{{ policy.type | iamType }}</chef-table-cell>
                       <chef-table-cell class="status-column">{{ status(policy) | capitalize }}</chef-table-cell>
                       <chef-table-cell class="three-dot-column">
-                      <!-- <app-authorized [allOf]="['/policies/{policy}', 'delete', policy]"> -->
+                      <!-- <app-authorized [allOf]="[`/iam/v2beta/policies/{id}`, 'delete', policy.id]"> -->
```

Comment:
PR Bonus: Added relevant consistency cleanup in this PR for the **_non_**-project pages.
Here, you will note, before and after are both commented out, but the content has been corrected.

Reply:
Can we just delete this commented code for now? We don't even know if this is what the final form will be.